package.json

```diff
@@ -47,6 +50,7 @@
     "@cosmjs/tendermint-rpc": "^0.29.4",
     "@skip-mev/skipjs": "^1.1.0",
```

Comment:
Any interest in adding #54 to update skip and cosm?  Or wait until a future release?

Reply:
I have a question out with skip how to solve this signer issue. I'd really rather not use ts-ignore to get a private signer out.

Reply:
Oh good, I would prefer they fixed their issue as well.  Thanks for the update!